src/conspiracies/pipeline/pipeline.py

```diff
@@ -106,11 +112,22 @@ def corpusprocessing(self):
                         for field in ("subject", "predicate", "object")
                     ]
                     print(", ".join(triplet_fields), file=out)
+                    n_triplets += 1
+
+        if self.config.corpusprocessing.auto_thresholds:
+            thresholds = ClusteringThresholds.auto(n_triplets)
+        else:
+            thresholds = self.config.corpusprocessing
+
         umap_hdb.main(
             f"{self.output_path}/triplets.csv",
-            "danskBERT",
-            dim=40,
+            "danish" if self.config.base.language == "da" else "english",
```

Comment:
Assuming we want to support non-english as well, I would probably just make this into a dictionary.

Reply:
Yeah, that's a matter of preference. I see both work. For a two-way branching I find the inline if-else more concise than a dict, and I don't know of the project supporting more than two languages in the near future.